core/src/main/resources/logback.xml

```diff
@@ -2,7 +2,20 @@
 
     <appender name="STDOUT" class="ch.qos.logback.core.ConsoleAppender">
         <encoder>
-            <pattern>%d{HH:mm:ss.SSS} [%thread] - %msg%n</pattern>
+            <pattern>%d{HH:mm:ss.SSS} [%thread] %logger{35} - %msg%n</pattern>
+        </encoder>
+    </appender>
+
+    <appender name="FILE" class="ch.qos.logback.core.FileAppender">
+        <file>/cyberdata/cm-connector.log.txt</file>
+        <rollingPolicy class="ch.qos.logback.core.rolling.SizeAndTimeBasedRollingPolicy">
+            <fileNamePattern>log-%d{yyyy-MM-dd}.%i.txt</fileNamePattern>
+            <maxFileSize>10MB</maxFileSize>
+            <maxHistory>30</maxHistory> <!-- max 30 days-->
+            <totalSizeCap>1GB</totalSizeCap> <!-- max 1GB-->
+        </rollingPolicy>
```

Comment:
This one is nice. Are there any options to limit disk writes per second? Just curious.